app/channel.go

```diff
@@ -1330,13 +1325,11 @@ func (a *App) addUserToChannel(user *model.User, channel *model.Channel, teamMem
 
 	newMember, nErr = a.Srv().Store.Channel().SaveMember(newMember)
 	if nErr != nil {
-		mlog.Error("Failed to add member", mlog.String("user_id", user.Id), mlog.String("channel_id", channel.Id), mlog.Err(nErr))
-		return nil, model.NewAppError("AddUserToChannel", "api.channel.add_user.to.channel.failed.app_error", nil, "", http.StatusInternalServerError)
+		return nil, model.NewAppError("AddUserToChannel", "api.channel.add_user.to.channel.failed.app_error", map[string]interface{}{"user_id": user.Id, "channel_id": channel.Id}, "", http.StatusInternalServerError)
```

Comment:
IIUC, you will also have to change the error string in en.json to include {{user_id}} and {{channel_id}} as template params, for these to show up.

Reply:
Hmm, that's correct. The idea is to keep information on the error struct. Still, there is no point if we are not going to print them.